app/mailers/application_mailer.rb

```diff
@@ -1,4 +1,11 @@
 class ApplicationMailer < ActionMailer::Base
   default from: "no-reply@#{Rails.configuration.default_email_domain}"
   layout 'mailer'
+
+  def add_env_if_needed(email_subject)
+    env = Rails.configuration.cmr_env.upcase
+    if env != 'OPS'
+      email_subject = email_subject + " (#{env})"
```

Comment:
1. I'm not sure why we this is checking against the `cmr_env` instead of what MMT is running.
2. I don't think the assignment on line 8 is necessary, and I think the preferred syntax is to put it all in quotes and use interpolation
So I think lines 6 - 8 can be replaced with
```
unless Rails.env.production?
  "#{email_subject} (#{Rails.env})"
```

Reply:
Because we don't want any change for production env

Reply:
is your comment about the "unless" code (question 2) or question 1?

Reply:
It was about 2, but it is clear now, nil will not hurt